docs/source/osds/scenarios.rst

```diff
@@ -171,185 +196,204 @@ An example of using the ``non-collocated`` OSD scenario with encryption, bluesto
+and journals would be placed on the faster solid state device ``/dev/nvme0n1``.
 
-.. code-block:: yaml
+Other (optional) supported settings:
 
-   osd_objectstore: filestore
-   osd_scenario: lvm
-   lvm_volumes:
-     - data: data-lv1
-       data_vg: vg1
-       journal: journal-lv1
-       journal_vg: vg2
-       crush_device_class: foo
-     - data: data-lv2
-       journal: /dev/sda
-       data_vg: vg1
-     - data: data-lv3
-       journal: /dev/sdb1
-       data_vg: vg2
-     - data: /dev/sda
-       journal: /dev/sdb1
-     - data: /dev/sda1
-       journal: journal-lv1
-       journal_vg: vg2
+- ``crush_device_class``: Sets the crush class for all OSDs created with this
+  method (it is not possible to have a per-OSD crush class using the *simple*
+  configuration approach)
 
-For example, a configuration to use the ``lvm`` osd scenario with encryption would look like:
 
-.. code-block:: yaml
+Advanced configuration
+^^^^^^^^^^^^^^^^^^^^^^
 
-   osd_objectstore: filestore
-   osd_scenario: lvm
-   dmcrypt: True
-   lvm_volumes:
-     - data: data-lv1
-       data_vg: vg1
-       journal: journal-lv1
-       journal_vg: vg2
-       crush_device_class: foo
+This configuration is useful when more granular control is wanted when setting
+up devices and how they should be arranged to provision an OSD. It requires
+initial setup of the devices with LVM, and optionally partitions for journals,
+or ``block.db`` and ``block.wal``.
 
-If you wished to use ``devices`` instead of ``lvm_volumes`` your configuration would look like:
+To use this configuration, the ``lvm_volumes`` option must be populated with
+logical volumes, volume groups, and optionally partitions for journals, or
+``block.db`` and ``block.wal``.
 
-.. code-block:: yaml
+.. note:: This configuration uses ``ceph-volume lvm create`` to provision OSDs
 
-   osd_objectstore: filestore
-   osd_scenario: lvm
-   crush_device_class: foo
-   devices:
-     - /dev/sda
-     - /dev/sdc
+Supported ``lvm_volumes`` configuration settings:
+
+- ``data``: The logical volume name or full path to a raw device (an LV will be
+  created using 100% of the raw device)
+
+- ``data_vg``: The volume group name. If using a logical volume name for ``data``
+    it is **required**.
+
+- ``crush_device_class``: Device class name for the resulting OSD, allows to
+  individually set the class for each OSD, unlike the global
+  ``crush_device_class`` that sets them for all OSDs.
+
+.. note:: If you wish to change set the ``crush_device_class`` for the OSDs
+          when using ``devices`` you must set it using the global ``crush_device_class``
+          option as shown above. There is no way to define a specific crush device class
+          per OSD when using ``devices`` like there is for ``lvm_volumes``.
+
+
+``filestore`` objectstore only values:
+
+- ``journal``: The logical volume name or full path to a raw device (an LV will be
+  created using 100% of the raw device)
```

Comment:
I thought journal needed a partition and not a raw device?

Reply:
this is correct, need to fix this